docs/superpowers/specs/2026-08-28-schema-ea-executive-ontology-spec.md

```
@@ -0,0 +1,297 @@
+| `handoffFor` | handoff → service | Handoff is for this service. |
+| `supersedes` | creativework → creativework | This document replaces an older one. |
+| `hasRole` | person → role | Person fills this role. |
+| `operates` | role → software_application | Role is responsible for this product. |
+| `provides` | organization → service | Organization provides this service. |
+| `maintains` | person → software_application | Person maintains this product. |
+
+---
+
+## Package Structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)